core/src/lib/tls_openssl_private.cc

```diff
@@ -145,6 +145,8 @@ bool TlsOpenSslPrivate::init()
 
   SSL_CTX_set_options(openssl_ctx_, SSL_OP_NO_SSLv2 | SSL_OP_NO_SSLv3);
 
+  SSL_CTX_set_options(openssl_ctx_, SSL_OP_CIPHER_SERVER_PREFERENCE);
```

Comment:
I'm not completely sure we want to have that enabled unconditionally.

Reply:
Isn't that an advantage, if for some reason another client daemon want to connect with a larger choice ?

Reply:
Commit dropped.